src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGeneratorTest.java

```diff
@@ -300,12 +306,99 @@ public void testFirstImport() {
     @Test
     public void testLastImport() {
         final int lineNumber = 8;
-        final int columnNumber = 0;
+        final int columnNumber = 1;
         final XpathQueryGenerator queryGenerator = new XpathQueryGenerator(rootAst, lineNumber,
-                columnNumber);
+                columnNumber, fileName, DEFAULT_TAB_WIDTH);
         final List<String> actual = queryGenerator.generate();
         final List<String> expected = Collections.singletonList(
                 "/IMPORT[./DOT[@text='Iterator']]");
         assertEquals("Generated queries do not match expected ones", expected, actual);
     }
+
+    @Test
+    public void testTabWidthOne() throws Exception {
```

Comment:
We can't name these methods something better than 1,2,3,4 ?

Reply:
@timurt 
Be more descriptive in names. The test name should have the short description of its purpose in it.

Reply:
done